tracer/src/Datadog.Trace/Ci/Test.cs

```diff
@@ -347,6 +347,17 @@ public void Close(TestStatus status, TimeSpan? duration, string? skipReason)
             case TestStatus.Skip:
                 tags.Status = TestTags.StatusSkip;
                 tags.SkipReason = skipReason;
+                if (tags.SkipReason == TestTags.SkippedByIntelligentTestRunnerReason)
+                {
+                    tags.SkippedByIntelligentTestRunner = "true";
+                    Suite.Tags?.IncrementIntelligentTestRunnerSkippingCount();
```

Comment:
Not sure if I'm reading it right, but does `Suite.Tags?` mean we're incrementing the count in suite? If so, is it done just for customers' convenience? Asking since the RFC does not mention this

Reply:
Yes, only for customer's convenience, it feels weird we have an indication at test level with the boolean flag, and a count at module level, but nothing at suite level.